### PR TITLE
Add azure test conf

### PR DESCRIPTION
**Pull request by TheoPascoli**
**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*

**What is the current behavior?** *(You can also link to an open issue here)*

**What is the new behavior (if this is a feature change)?**

**Does this PR introduce a breaking change?** *(What changes might users need to make in their application due to this PR?)*

**Other information**:

(if any of the questions/checkboxes don't apply, please delete them entirely)

### CODE REVIEW

File: k8s/overlays/azure/test/kustomization.yaml [hunk 1]
```diff
@@ -77,6 +79,14 @@ patches:
               - name: cse-import-d2cc-runner-itools-configmap-volume
                 configMap:
                   name: cse-runner-itools-azure-test-configmap
+  # Scale-up to 4 instances and adapt CSE runner itools config
```
Comment (WalAmeni) [inline k8s/overlays/azure/test/kustomization.yaml:82]: Please check that the computation for one task will not be runned 4 times

File: k8s/overlays/azure/test/kustomization.yaml [hunk 2]
```diff
@@ -16,6 +17,7 @@ resources:
   - cse-gridcapa-runner-itools-azure-test-configmap.yaml
   - cse-import-d2cc-app-idp-settings-configmap.yaml
   - cse-import-idcc-app-idp-settings-configmap.yaml
+  - swe-d2cc-app-idp-settings-configmap.yaml
```
Comment (WalAmeni) [inline k8s/overlays/azure/test/kustomization.yaml:20]: Could you add a specific itools config for swe to use xpress only on Azure test

File: k8s/bases/swe-d2cc/swe-d2cc-exporter.yaml [hunk 3]
```diff
@@ -98,12 +98,12 @@ spec:
             - name: SOURCE_USER
               valueFrom:
                 secretKeyRef:
-                  name: gridcapa-ftp-server-credentials
+                  name: gridcapa-ftp-credentials
```
Comment (WalAmeni) [inline k8s/bases/swe-d2cc/swe-d2cc-exporter.yaml:101]: Should use the gridcapa-ftp-server-credentials for azure ftp

**Review by WalAmeni — CHANGES_REQUESTED** [review]

File: k8s/overlays/azure/test/kustomization.yaml [hunk 4]
```diff
@@ -77,6 +79,14 @@ patches:
...
                 configMap:
                   name: cse-runner-itools-azure-test-configmap
+  # Scale-up to 4 instances and adapt CSE runner itools config
+  - patch: |-
+      kind: Deployment
+      apiVersion: apps/v1
+      metadata:
+        name: swe-d2cc-runner
+      spec:
+        replicas: 4
```
Comment (WalAmeni) [inline k8s/overlays/azure/test/kustomization.yaml:90]: Could you put replicas to 1 until the bug of parallelization is fixed ?

**Review by WalAmeni — CHANGES_REQUESTED** [review]

File: k8s/overlays/azure/test/kustomization.yaml (hunk 4 above)
Comment (TheoPascoli) [reply k8s/overlays/azure/test/kustomization.yaml:90]: It is fixed

File: k8s/overlays/azure/test/kustomization.yaml (hunk 1 above)
Comment (TheoPascoli) [reply k8s/overlays/azure/test/kustomization.yaml:83]: done

**Review by WalAmeni — APPROVED** [review]

**Pull request merged**